### ISSUE TITLE
The extension makes top bar menus larger than normal

### ISSUE THREAD

**Issue by seldon1000** [2nv2u/gnome-shell-extension-syncthing-indicator#3]
This is strange actually. I've had this issue for some time, but I realized just now that Syncthing Indicator is responsible. When the extension is enabled, all the top bar menus become larger: places, system menu, app menu...

An example below:
(extension disabled)
![Schermata del 2021-04-03 22-32-07](https://user-images.githubusercontent.com/55358113/113490824-f9326200-94cc-11eb-8ddd-9767c5fd163d.png)

(extension enabled)
![Schermata del 2021-04-03 22-32-47](https://user-images.githubusercontent.com/55358113/113490844-07807e00-94cd-11eb-9411-b3bf4c0bd9ae.png)

This is on GNOME 40, but the issue was there on 3.38 too.

**Comment by 2nv2u** [comment]
Interesting indeed, which extension are you presenting in the images?

**Comment by seldon1000** [comment]
That one is the integrated Places Status Indicator, from GNOME itself. Here are other screenshots (before/after).

System menu:
![photo_2021-04-04_08-56-22](https://user-images.githubusercontent.com/55358113/113501158-a9cb5080-9523-11eb-9775-a3b615370024.jpg)
![photo_2021-04-04_08-56-26](https://user-images.githubusercontent.com/55358113/113501162-acc64100-9523-11eb-9c5a-d41351066897.jpg)


Telegram menu (from KStatusNotifier extension):
![Schermata del 2021-04-04 08-50-19](https://user-images.githubusercontent.com/55358113/113501204-ce272d00-9523-11eb-9895-09c25ff5789a.png)
![Schermata del 2021-04-04 08-52-00](https://user-images.githubusercontent.com/55358113/113501211-d1bab400-9523-11eb-97d7-34dd65582686.png)


Nextcloud menu (from KStatusNotifies):
![Schermata del 2021-04-04 08-50-38](https://user-images.githubusercontent.com/55358113/113501229-de3f0c80-9523-11eb-86ea-006273149fc0.png)
![Schermata del 2021-04-04 08-52-21](https://user-images.githubusercontent.com/55358113/113501292-36760e80-9524-11eb-8458-f75751e30c86.png)


App menu:
![Schermata del 2021-04-04 08-51-05](https://user-images.githubusercontent.com/55358113/113501253-ee56ec00-9523-11eb-91d8-83b92ae2b4af.png)
![Schermata del 2021-04-04 08-52-36](https://user-images.githubusercontent.com/55358113/113501255-f0b94600-9523-11eb-84d7-e6fbb2948d1f.png)


Basically, any menu from the top bar, be it system or an extension, is larger than normal. Disabling any other extension doesn't change anything.

**Comment by 2nv2u** [comment]
I think it's fixed now, can you test it by installing the extension from here?
(Do not switch the autostart function on/off, it will overwrite the systemd config for version 1.15 which isn't released yet)

**Comment by seldon1000** [comment]
It's fixed! Thank you.

**Comment by 2nv2u** [comment]
Great! It will be part of the next release then, I'm waiting for syncthing 1.15 though.
